src/test/integration_tests/api/iam/test_iam_basic_integration.js

```diff
@@ -32,33 +33,39 @@ const setup_options = { should_run_iam: true, https_port_iam: 7005, debug: 5 };
 coretest.setup(setup_options);
```

Comment:
I think you need to change here, as some of the options are related to NC and not containerized.